src/Goerli-DssSpell.sol

```diff
@@ -46,6 +72,13 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
         return false;
     }
 
+    bytes32 internal constant ILK = "TELEPORT-FW-A";
+    bytes32 internal constant DOMAIN_ETH = "ETH-GOER-A";
+    bytes32 internal constant DOMAIN_STA = "STA-GOER-A";
+    address internal constant TELEPORT_GATEWAY_STA = 0x140f746CcFbb1C2618838C063048949685d7A6eD;
+    uint256 internal constant TELEPORT_L2_GATEWAY_STA = 0x042b46146f0a377e0a028ed44bc1c0567196b8b96f3c7ab469e593ca497e2a83;
```

Comment:
Looking into the events on this starknet contract there seems to be 2 relys, but no denies.
Would expect the deployer to be denied:
https://goerli.voyager.online/contract/0x042b46146f0a377e0a028ed44bc1c0567196b8b96f3c7ab469e593ca497e2a83#events

Reply:
This was now done.
Regarding what each rely and deny does:

On goerli: https://testnet.starkscan.co/contract/0x042b46146f0a377e0a028ed44bc1c0567196b8b96f3c7ab469e593ca497e2a83#events:
* first rely is the deployer
* second rely is the governance relay: 0x30255465a3d33f430ea6e16cb22cc09b9291972f7f8c7198b5e5b1ef522b85c, you can check its address in the corresponding L1Relayer: https://goerli.etherscan.io/address/0x73c0049dd6560e644984fa3af30a55a02a7d81fb#readContract#F1

Reply:
resolving this.